### PR TITLE
Add a state for when the clock runs out

### PR DESCRIPTION
**Pull request by jaxbot**
screen shot 2016-01-13 at 6 15 33 pm

### CODE REVIEW

File: frontend/_src/assets/js/_scripts.js [hunk 1]
```diff
@@ -1,16 +1,22 @@
 function updateCountdownClock() {
     var curTime = (new Date()).getTime();
     // Fri, 15 Jan 2016 GMT-5
-    var destTime = Date.parse("Sat, 16 Jan 2016 01:00:00 GMT");
+    var destTime = Date.parse("Sat, 12 Jan 2016 01:00:00 GMT");
```
Comment (iveysaur) [inline frontend/_src/assets/js/_scripts.js]: come on

Comment (maxcell) [inline frontend/_src/assets/js/_scripts.js]: ONE DAMN JOB

**Comment by iveysaur** [comment]
LGTM now

**Comment by iveysaur** [comment]
I see @maxcell is looking at it too so I'll give him time to look it over :)

**Comment by iveysaur** [comment]
@jaxbot is making a meme now we have to hurry

**Pull request merged**